### ISSUE TITLE
Paths mapping not working for declarations

### ISSUE THREAD

**Issue by anion155** [wessberg/rollup-plugin-ts#31]
# Expected to happen

Creation of declarations bundle.

# Actually happen

All the export's and import's with path mapping stayed unchanged and unresolver.

# Reproduce

See example repo [here](https://github.com/anion155/rollup-plugin-ts-decl-paths-mapping-bug).

**Comment by anion155** [comment]
@wessberg

**Comment by wessberg** [comment]
@anion155, thanks for writing. There's already an open issue for this: https://github.com/wessberg/rollup-plugin-ts/issues/14. Thanks for the repro, though.

I've seen the issue and will look into it when I have some time available.

**Comment by anion155** [comment]
I did investigate it a bit. With paths bindings set, when it comes to [this transformers](https://github.com/wessberg/rollup-plugin-ts/blob/master/src/service/transformer/declaration-bundler/declaration-bundler.ts), this code misses module imprted as binded path:
```typescript
if (!options.localModuleNames.includes(normalize(sourceFile.fileName))) return updateSourceFileNode(sourceFile, [], true);
```
Cause `sourceFile.fileName` is a relative path to `baseUrl` and localModuleNames has absolute paths in it.

**Comment by anion155** [comment]
At least it's a first think that stops bundle from generation.

**Comment by anion155** [comment]
Worth to mention, in my tests `originalFileName` was the absolute path.

**Comment by wessberg** [comment]
Thanks for reporting this issue. It has been fixed in v[v1.1.74](https://github.com/wessberg/rollup-plugin-ts/releases/tag/v1.1.74) that now has first-class support for path mapping.
